gamebench_api_client/global_settings.py

```diff
@@ -3,8 +3,8 @@
 GAMEBENCH_CONFIG = {
     'url': 'https://api.production.gamebench.net',
     'api_version': '/v1',
-    'username': '',
-    'password': '',
+    'username': 'john.smith',
```

Comment:
Are these placeholder values necessary in the production code?

Reply:
No, I just thought placeholder might help the visual. I will remove it to avoid any later confusion!

Reply:
I'm ready to approve once this change is pushed.

Reply:
Removed in 0a40931

Reply:
Thanks! I wonder why it isn't showing in the "view changes" view? I went to the file directly to confirm.